### PR TITLE
update azure iot example

### PR DESCRIPTION
**Pull request by oysteingr**

### CODE REVIEW

File: examples/azure_iot/README.md [hunk 1]
```diff
@@ -1,25 +1,197 @@
...
+
+It requires two environment variables to be set in order to communicate with Azure, `PROVISIONING_ID_SCOPE` and `PROVISIONING_SYMMETRIC_KEY`.
+
+And here’s screenshots of what it looks like a after successful provisioning.
+
+### Provisioning Service
+![image](https://user-images.githubusercontent.com/3704622/58804717-d013c280-8612-11e9-9c05-05ec4e7e0835.png)
+
+### IoT Hub
+![image](https://user-images.githubusercontent.com/3704622/58804753-e883dd00-8612-11e9-8dbd-91b622909fd0.png)
```
[the remark this answers is not in the document]
Comment (jontore) [reply examples/azure_iot/README.md]: Also missing ```  await eventHandler.init();``` when starting the service.

File: examples/azure_iot/README.md [hunk 2]
```diff
@@ -1,25 +1,197 @@
...
+    console.log('Woho, we have a device. ' +
+        `Serial Number ${info.serialNumber} ` +
+        `Software Version: ${info.version}`);
+    process.exit(0);
+});
+
+huddlySdk.init();
+```
+
+Now that we have communication with the device up and running, lets start retrieving some more intelligent data from the camera. The camera has an onboard neural compute engine and is programmed to performed convolutional operations on full-image frames. The current neural network on Huddly IQ is optimized for detecting people and heads.
```
Comment (brikendr) [inline examples/azure_iot/README.md]: `[](url)`
```suggestion
Now that we have communication with the device up and running, let's start retrieving some more intelligent data from the camera. The camera has an onboard neural compute engine and is programmed to perform convolutional operations on full-image frames. The current neural network on Huddly IQ is optimized for detecting people and heads. 
```

File: examples/azure_iot/README.md [hunk 3]
```diff
@@ -1,25 +1,197 @@
...
+![image](https://user-images.githubusercontent.com/3704622/58803982-03ede880-8611-11e9-80a1-89a2729abacd.png)
+
+### Technical overview
+
+![image](https://user-images.githubusercontent.com/3704622/58804343-eec58980-8611-11e9-83cc-8d821de719eb.png)
+
+
+# Getting up and running with the SDK
+
+Before we dive in to Azure and the analytics dashboard, we will start by getting up and running with the Huddly SDK. The Huddly SDK is a toolkit for NodeJS based applications to easily get started. The SDK itself is open source its code can be found on Github <Link>.
```
[the remark this answers is not in the document]
Comment (oysteingr) [reply examples/azure_iot/README.md]: :+1:

File: examples/azure_iot/README.md [hunk 4]
```diff
@@ -1,25 +1,197 @@
...
+
+Now lets get NodeJS and the Huddly SDK. The Huddly SDK uses a native NodeJS module to interact with the USB interface of the camera. This native module is currently compiled for NodeJS version 10. Since NodeJS is a rapidly changing environment and new versions coming out quite often, I prefer using a version manager to be able to toggle between multiple version. There are several of these, but currently I’m sticking to `nvm`.
```
[the remark this answers is not in the document]
Comment (oysteingr) [reply examples/azure_iot/README.md]: Right, fixing

File: examples/azure_iot/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (oysteingr) [reply examples/azure_iot/README.md]: Thanks

File: examples/azure_iot/README.md (hunk 2 above)
Comment (oysteingr) [reply examples/azure_iot/README.md]: thx

**Pull request merged**